packages/backend/src/clients/Slack/Slackbot.ts

```diff
@@ -74,27 +74,42 @@ export class SlackBot {
         }
 
         try {
-            const slackReceiver = new ExpressReceiver({
-                ...slackOptions,
-                installationStore: {
-                    storeInstallation: (i) =>
-                        this.slackAuthenticationModel.createInstallation(i),
-                    fetchInstallation: (i) =>
-                        this.slackAuthenticationModel.getInstallation(i),
-                    deleteInstallation: (i) =>
-                        this.slackAuthenticationModel.deleteInstallation(i),
-                },
-                logLevel: LogLevel.INFO,
-                app: expressApp,
-            });
-
-            const app = new App({
-                ...slackOptions,
-
-                receiver: slackReceiver,
-            });
+            if (this.lightdashConfig.slack?.appToken) {
```

Comment:
@Shifan-Gu Let's introduce a `SLACK_SOCKET_MODE` env var and use it for this condition. Two reasons for this: explicit configuration, prevent backwards compatibility issues as old instances might still have `SLACK_APP_TOKEN` env var.

Reply:
sure, will do 👍